chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,73 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate Quick Copy-type menulist with locales
+	 * (for locale menulists that don't update with style selection)
+	 */
+	this.populateLocaleList = function(doc, menulistID, prefLocale) {
```

Comment:
You can just pass the menulist itself and use ``menulist.ownerDocument`` to get ``doc``.